darwin/dataset/upload_manager.py

```diff
@@ -125,6 +207,30 @@ def read(self, size: int = -1) -> Any:
 
 
 class UploadHandler:
+    """
+    Holds responsabilities for file upload management and failure into ``RemoteDataset``s.
+
+    Parameters
+    ----------
+    dataset: RemoteDataset
+        Target ``RemoteDataset`` where we want to upload our files to.
+    local_files: List[LocalFile]
+        List of ``LocalFile``s to be uploaded.
+
+    Attributes
+    ----------
+    dataset : RemoteDataset
+        Target ``RemoteDataset`` where we want to upload our files to..
+    errors : List[UploadRequestError]
+        List of errors that happened during the upload process.
+    local_files: List[LocalFile]
```

Comment:
missing whitespace between `local_files` and colon

Reply:
Good catch !